### DISCUSSION TITLE
Grey out "Delete Search" button for empty string

### DISCUSSION THREAD

**Discussion by stabilergriller** [quodlibet/quodlibet#4229] [Ideas]
Just a small suggestion. How about greying out the arrow pointing left in the search field, when there's no term entered?

This is how it looks like in the moment:

![Screenshot_2022-12-25_16-52-24](https://user-images.githubusercontent.com/115423284/209474563-9e65e19f-d53e-47f4-93ce-78444305b767.png)

Mock up for my idea:

![Screenshot_2022-12-25_16-56-00](https://user-images.githubusercontent.com/115423284/209474628-f4b8ee14-ec24-4e48-9d93-03f24d3a84da.png)

(This is not something I did myself, but rather a bug that occurs when switching between light / dark version of a theme in the plugins section - I think)

So after entering one / several characters in empty field => turn icon from grey to black
When deleting character(s) => turn icon from black to grey, if no chars left

Of course, one must also account for dark themes.

Tested on Pop!_OS 22.04 LTS (Live CD) with QL installed from Repository (not Flatpak)

**Comment by declension** [comment]
Converted to #4234
